pyproject.toml

```diff
@@ -46,24 +46,25 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Visualization",
 ]
 dependencies = [
-    "anndata>=0.7.4",
+    "anndata>=0.8",
     # numpy needs a version due to #1320
-    "numpy>=1.17.0",
+    "numpy>=1.23",
     "matplotlib>=3.6",
-    "pandas >=2.1.3",
-    "scipy>=1.4",
-    "seaborn>=0.13.0",
-    "h5py>=3",
+    "pandas >=1.5",
+    "scipy>=1.8",
+    "seaborn>=0.13",
+    "h5py>=3.1",
     "tqdm",
     "scikit-learn>=0.24",
-    "statsmodels>=0.10.0rc2",
+    "statsmodels>=0.13",
     "patsy",
-    "networkx>=2.3",
+    "networkx>=2.7",
     "natsort",
     "joblib",
-    "numba>=0.41.0",
+    "numba>=0.56",
     "umap-learn>=0.3.10",
-    "packaging",
+    "pynndescent>=0.5",
+    "packaging>=21.3",
```

Comment:
This is calver, but they have never issued more than one release per month so it doesn’t matter.